services/QuillLMS/lib/query_examples/snapshots/9874030-2023-08-01-2023-12-01/active-classrooms.sql

```diff
@@ -0,0 +1,26 @@
+      /* Data Processed By Query: 0.53 GB */
```

Comment:
+1 to gitignoring these files, eventually. They take up space and will eventually drift from what's in production.